Stratis.Bitcoin/Consensus/ConsensusFeature.cs

```diff
@@ -29,6 +29,7 @@ public class ConsensusFeature : FullNodeFeature
 		private readonly Signals signals;
 		private readonly ConsensusLoop consensusLoop;
 		private readonly NodeSettings nodeSettings;
+		private readonly StakeChain stakeChain;
```

Comment:
Its definitely possible but Its becoming too messy don't you think?

In fact I am trying to avoid naming coins but rather name the technology we should even rename StratisConsensusValidator to maybe POSConsensusValidator and POWConsensusValidator (for bitcoin) POS will inherit from POW as its built on top of it. then the only way to distinguish a network is the Network builder and the ConsensusOptions (even that should maybe incorporated to Network but as Network is inside NBitcoin we can't for now)

If a coin is diverting then the default validators then yeah sure make an override.